python/plugins/processing/gui/AlgorithmDialog.py

```diff
@@ -340,6 +341,7 @@ def on_complete(ok, results):
 
     def finish(self, successful, result, context, feedback, in_place=False):
         keepOpen = not successful or ProcessingConfig.getSetting(ProcessingConfig.KEEP_DIALOG_OPEN)
+        HtmlOutputs = False
```

Comment:
Can you open a separate PR for this fix? It's unrelated to the grass ones

Reply:
Thanks! Can you update this PR and remove that commit now?

Reply:
I've reverted, squashed and force-pushed.  Hopefully that's the proper way to do this!